### ISSUE TITLE
Restorative Totems talent doesn't increase Mana Spring Totem regeneration.

### ISSUE THREAD

**Issue by Momot98** [dalaranwow/dalaran-wow#5669]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I'm a Restoration Draenai Shaman, and I put three points into the Restorative Totems talent, which increases the effect of the Mana Spring Totem by 20% and the Healing Stream Totem by 45%. With all 3 points in this talent, it says 25 mana should be restored by the Mana Spring Totem every 5 seconds. However, when I actually put the totem down, it only restores 21 mana every 5 seconds.

**Expected behaviour**: If the talent were working correctly, it should be applying the 20% increase in mana regeneration. It seems to say it applies it, but the final totem doesn't actually.

**Steps to reproduce the problem**:

1. Put 3 talents into Restorative Totems in the Restoration talent tree
2. Put down the Mana Spring Totem and see if it regenerates as much mana as it's supposed to

**Include proofs for this behaviour**

The talent:
https://i.imgur.com/AxHRL1A.png

The amount it should regenerate:
https://i.imgur.com/dvnYG0a.png

The amount it actually does regenerate:
https://i.imgur.com/SA2GYIL.png

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
This is same case as on retail, where the buff & debuff tooltips don't take into account any talents/glyphs, so they will always show the default amount.

You can check the correct amount in your action bar/spellbook tooltip. Thanks

Closing.
